### PR TITLE
Int signed instances

### PR DESCRIPTION
**Pull request by proux01**
##### Motivation for this change

Makes positive constants posnum (for instance 1 / 3 is now a posnum, following yesterday's meeting discussion: https://github.com/math-comp/analysis/wiki/2023-07-27-Meeting )

##### Things done/to do

- [x] added corresponding entries in `CHANGELOG_UNRELEASED.md`

- [ ] ~added corresponding documentation in the headers~

##### Compatibility with MathComp 2.0

- [x] I added the label `TODO: HB port` to make sure someone ports this PR to
      the `hierarchy-builder` branch **or** I already opened an issue or PR (please cross reference).

##### Automatic note to reviewers

Read [this Checklist](https://github.com/math-comp/math-comp/wiki/Checklist-for-creating-and-review-PRs) and put a milestone if possible.

### CODE REVIEW

**Comment by proux01** [comment]
Pinging @zstone1 @CohenCyril and @affeldt-aist

File: CHANGELOG_UNRELEASED.md [hunk 1]
```diff
@@ -168,6 +172,13 @@
 - in `topology.v`:
   + lemma `my_ball_le` (use `ball_le` instead)
 
+- in `signed.v`:
+  + lemma `nat_snum_subproof`
+  + canonical instance `nat_snum` (uselles, there is already a default instance
```
Comment (zstone1) [inline CHANGELOG_UNRELEASED.md:177]: Uselles -> useless

**Review by zstone1 — APPROVED** [review]
Minor typo. Otherwise seems straightforward.

**Review by CohenCyril — APPROVED** [review]

**Comment by proux01** [comment]
CI "green"

**Pull request merged**